### PR TITLE
IGAPP-1284: Fix transparent language dropdown

### PR DESCRIPTION
**Pull request by steffenkleinle**
This pull request belongs to an issue on our [bugtracker](https://issues.integreat-app.de/).
You can find it there by looking for an issue with the key which is mentioned in the title of this pull request.
It starts with the keyword **IGAPP**.

### CODE REVIEW

File: web/src/components/HeaderActionItemDropDown.tsx [hunk 1]
```diff
@@ -35,6 +34,7 @@ export const Container = styled.div`
     object-fit: contain;
     width: 100%;
     height: 100%;
+    opacity: 0.85;
```
Comment (f1sh1918) [inline web/src/components/HeaderActionItemDropDown.tsx:37]: can you give me a hint why this is needed?

**Review by f1sh1918 — APPROVED** [review]
transparency is fixed. thx for that. couldn't find the need for this opacity you set

**Review by sarahsporck — APPROVED** [review]
And here I was and thought this was intended ^^
Tested in Firefox.

File: web/src/components/HeaderActionItemDropDown.tsx (hunk 1 above)
Comment (steffenkleinle) [reply web/src/components/HeaderActionItemDropDown.tsx:37]: just to make the language change icon look more grey and less black I guess, should be fixed in https://issues.tuerantuer.org/browse/IGAPP-1287

**Pull request merged**